did_resolver_registry/src/error.rs

```diff
@@ -3,12 +3,16 @@ use std::error::Error;
 #[derive(Debug)]
 pub enum DidResolverRegistryError {
     UnsupportedMethod,
+    UnqualifiedDid,
```

Comment:
in context of did resolver, I think it would be safe to assume that unqualified did are `did:sov:<unqialified>`?

Reply:
I don't think so. If you know you have unqualified did, you should qualify it before attempting to resolve it (with the method only you know you want to use, it is not implicit). If you don't know you have unqualified DID, you should get a clear error indicating such fact instead of failing later with potentially confusing error unrelated to the root cause.

Reply:
fair enough